R/logger.R

```diff
@@ -29,12 +29,38 @@ setup_logger <- function(output_dir, log_name) {
 
 #' @title Add a basic default file logger for a newly processed file.
 #'
-#' @param output_root Output root directory for the current process.
 #' @param logfile The name of the log file.
-setup_file_logger <- function(output_root, logfile) {
+#' @param output_root Output root directory for the current process.
+#'
+#' @return returns the loggers that where previously set for usage with with_
+setup_file_logger <- function(logfile, output_root) {
+    loggers <- getLoggers()
+    clearLoggers()
```

Comment:
why do we have to clear loggers again here if that is done already by the wrapper function `with_file_logger` before the `setup_file_logger` function is called?